charts/datadog/values.yaml

```diff
@@ -254,6 +254,19 @@ datadog:
   ## ref: https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#environment-variables
   checksCardinality:  # low, orchestrator or high (not set by default to avoid overriding existing DD_CHECKS_TAG_CARDINALITY configurations, the default value in the Agent is low)
 
+  # SBOM configuration
```

Comment:
If you can add also `datadog.containerImage.collection.enabled` to enabled that containerImage metadata collection (useful to SBOM to display info on the container image page)